### PR TITLE
feat(telecom): add line configuration modal

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-14099

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat#MANAGER-14099
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
~~- [ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.routing.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ export default /* @ngInject */ ($stateProvider) => {
         },
       },
       resolve: {
+        isConfiguration: /* @ngInject */ ($transition$) =>
+          $transition$.params().isConfiguration,
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.routing.js:13]:
```suggestion
        deviceId: /* @ngInject */ ($transition$) =>
          $transition$.params().deviceId,
```
Maybe you can use the deviceId as params ? If you have it's a configuration if not it's new add device

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html [hunk 2]
```diff
@@ -110,7 +110,9 @@ <h4 class="pt-3">
                             >
                             </span>
                         </oui-button>
-                        <oui-button class="p-2"
+                        <oui-button
+                            class="p-2"
+                            data-on-click="$ctrl.openAddDeviceModal(true)"
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html:115]:
```suggestion
                            data-on-click="$ctrl.openAddDeviceModal(deviceId)"
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js [hunk 3]
```diff
@@ -17,9 +17,10 @@ export default /* @ngInject */ ($stateProvider) => {
           $translate.instant(
             'telephony_line_phone_actions_line_details_softphone_breadcrumb',
           ),
-        openAddDeviceModal: /* @ngInject */ ($state) => () =>
+        openAddDeviceModal: /* @ngInject */ ($state) => (isConfiguration) =>
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js:20]:
```suggestion
        openAddDeviceModal: /* @ngInject */ ($state) => (deviceId) =>
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js [hunk 4]
```diff
@@ -17,9 +17,10 @@ export default /* @ngInject */ ($stateProvider) => {
...
           $state.go(
             'telecom.telephony.billingAccount.line.dashboard.softphone.add',
+            { isConfiguration },
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js:23]:
```suggestion
            { deviceId },
```

**Review by ghyenne — CHANGES_REQUESTED** [review]

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -23,6 +24,7 @@
   "telephony_line_softphone_delete_device_success": "Votre softphone a bien été supprimé.",
   "telephony_line_softphone_add_device": "Ajouter un appareil",
   "telephony_line_softphone_add_my_device": "Ajouter mon appareil",
+  "telephony_line_softphone_configurate": "Configurer mon appareil",
```
Comment (sachinrameshn) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json:27]:
```suggestion
  "telephony_line_softphone_configure": "Configurer mon appareil",
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.html [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <oui-modal
     data-on-dismiss="$ctrl.goBack()"
-    data-heading="{{ ::'telephony_line_softphone_add_my_device' | translate }}"
+    data-heading="{{ $ctrl.deviceId ? 'telephony_line_softphone_configurate':'telephony_line_softphone_add_my_device' | translate }}"
```
Comment (sachinrameshn) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.html:3]:
```suggestion
    data-heading="{{ $ctrl.deviceId ? 'telephony_line_softphone_configure':'telephony_line_softphone_add_my_device' | translate }}"
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json (hunk 5 above)
Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json:27]: Sorry, Excuse my english

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**